user-interface/src/components/AssignAttorneyModal.tsx

```diff
@@ -16,7 +16,6 @@ export interface AssignAttorneyModalProps {
   attorneyList: Attorney[];
   bCase: Chapter15Type | undefined;
   modalId: string;
-  openerId: string;
```

Comment:
I'd love to hear the rationale of removing openerId, and why that was causing the issue.  Have we thought through all of the possible effects of removing this or perhaps why it was put in in the first place?